backend/server.mjs

```diff
@@ -14,7 +15,7 @@ dotenv.config({ path: envPath })
 
 const app = express()
 app.use(express.json())
-
+app.use(cors())
 // Enable CORS
```

Comment:
I think we can remove the commented out syntax if it's no longer needed.

Reply:
👍🏿 removed